tonic/src/transport/server.rs

```diff
@@ -207,7 +207,10 @@ impl Server {
                 #[cfg(feature = "tls")]
                 {
                     if let Some(tls) = &self.tls {
-                        let io = tls.connect(stream.into_inner()).await?;
+                        let io = match tls.connect(stream.into_inner()).await {
+                            Ok(io) => io,
+                            Err(_) => continue,
```

Comment:
What do you think about providing an error message here?

```suggestion
                            Err(error) => error!(message = "Unable to accept incoming connection.", %error);
```

Reply:
I'd be happy to - this has just introduced me to the `tracing` crate, thanks!